test/eval_test.cpp

```diff
@@ -36,6 +36,8 @@ struct id_target
     struct context
     {
         void finish() const {}
+        void finish_on(migraphx::any_ptr) const {}
+        void wait_for(migraphx::any_ptr) const {}
```

Comment:
Is this required anywhere in test file ?

Reply:
Can also be removed.